.github/workflows/ios-unsigned-device.yml

```
@@ -213,7 +326,145 @@ jobs:
+        if: ${{ always() }}
+        run: |
+          mkdir -p "$OUTPUT_DIR"
+          if [ -f build-log.txt ]; then
+            cp build-log.txt "$OUTPUT_DIR/" || true
+          fi
+
+      - name: Summarize xcodebuild log
+        if: ${{ always() }}
+        run: |
+          python3 <<'PY'
+from pathlib import Path
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 344: could not find expected ':'

(syntax-check)